API/Hardware_API.md

```diff
@@ -311,11 +316,36 @@ var tessel = new Tessel({
    B: false,
  }
 });
+```
+
+**tessel.open:**
 
-tessel.open('A'); // A is now open
-tessel.open('B'); // B is now open
+Open power to one or both Tessel ports.
```

Comment:
"open" phrasing feels non-obvious to me. Can we add another phrase that states it differently to be sure the function is clear?